crates/sui-indexer/src/models/tx_indices.rs

```diff
@@ -110,10 +120,24 @@ impl TxIndex {
         Vec<StoredTxKind>,
     ) {
         let tx_sequence_number = self.tx_sequence_number as i64;
+        let tx_affected_addresses = self
+            .recipients
+            .iter()
+            .chain(self.payers.iter())
```

Comment:
I wish there are more comments in the TxIndex data structure but my understanding is that recipients should naturally include payers?

Reply:
I'm not sure this is true, for the same reason that a sender (in the non-sponsorship case) is not necessarily in the `recipients` list: If the transaction transfers the gas coin to some other address.